### DISCUSSION TITLE
Can't call my route API

### DISCUSSION THREAD

**Discussion by Nevgup** [adonisjs/core#1800] [Help]
Hello

I created an application (web app, not an api) and I would like to call my post user route to create my resource from my minecraft java plugin (I have try with POSTMAN).
`POST: http://localhost:3333/api/users`
I have this error :
![image](https://user-images.githubusercontent.com/43917787/97329176-13f61000-1877-11eb-8473-6c1db6f02271.png)

Thanks in advance ! 😃

**Comment by trutimer** [comment]
Can you post your routes.js file? It would help

**Reply by Nevgup** [reply]
Yes my bad
```js
import Route from '@ioc:Adonis/Core/Route'

/**
 * AUTHENTICATION ROUTES
 */
Route.get('/signup', 'PagesController.index')
Route.get('/login', 'AuthController.show')

Route.group(() => {
    // Route.get('/', 'UsersController.index')
    Route.post('/', 'UsersController.store')
}).prefix('/api/users')


```

**Reply by trutimer** [reply]
From the HttpException it says Cannot GET. Which I assume you are trying to use GET method on a POST route. In your routes file you have GET commented. So you should use POST instead of get. Adonis tends to say ROUTE_NOT_FOUND instead of BAD_METHOD

**Reply by LeadcodeDev** [reply]
The problem is that when you use an API type route on a POST, Adonis sends you directly to the POST route without making a fuss.... Not in his case...
